ftw/crawler/configuration.py

```diff
@@ -15,6 +15,11 @@ def get_config(options):
         config.tika = options.tika
     if options.solr:
         config.solr = options.solr
+    # Command line options for SlackToken & SlackChannel override config values
```

Comment:
I would combine this into the comment above the whole block and make it a generic comment:
```
# Command line overrides for config options
```